components/Progress-Bar/progress-bar.js

```diff
@@ -0,0 +1,70 @@
+const msdsProgressBar = {
+  //Get the base number for the progress bar
+  parentElement: '',
+  baseNumber: 0,
+  currentStep: 0,
+  startFromOne: false,
+  init: function(parentElementId, baseNumber, customProgressionStep) {
+    this.parentElement = document.getElementById(parentElementId)
+
+    if (customProgressionStep > 0) {
+      this.currentStep = customProgressionStep
+    } else if (this.startFromOne == true) {
+      this.currentStep = 1
+    } else {
+      this.currentStep = 0
+    }
+    this.baseNumber = baseNumber
+    this.updateProgressbarDisplay()
+  },
+  updateProgressbarDisplay: function() {
+    //calculate the percentage width of the progress step and update the look and fill base on this
```

Comment:
This comment does not make sense for me? the only thing i see is that some math is applied to a number? where is the update happening?

Reply:
See the latest implementation I did today and the comment will make sense. we calculate the width and apply the new value to update the look and feel.